application/backend/src/main/java/pl/ds/bulma/components/models/TitleComponent.java

```diff
@@ -86,6 +110,12 @@ private void init() {
     }
     titleClasses = titleClassList.toArray(new String[]{});
     subtitleClasses = subtitleClassList.toArray(new String[]{});
+
+    ColorService colorService
+            = new ColorService(resource, "/libs/bulma/components/common/text/color",
```

Comment:
You can change the path to a relative one. It will give the possibility to overwrite it in the project easily.